tests/ha/control_pod/test_control_pod_restart.py

```diff
@@ -1683,3 +1631,214 @@ def test_chunk_upload_during_ctrl_pod_failover(self):
         LOGGER.info("Step 7: Successfully completed IOs.")
 
         LOGGER.info("ENDED: Test chunk upload during control pod restart")
+
+    # pylint: disable=too-many-branches
+    @pytest.mark.ha
+    @pytest.mark.lc
+    @pytest.mark.tags("TEST-45497")
+    def test_cruds_during_all_ctrl_pod_rst(self):
```

Comment:
in name it should be N_ctrl_pod_rst?

Reply:
Its given as all which is more generic than N I guess